src/dispatch/plugins/dispatch_slack/decorators.py

```diff
@@ -112,17 +168,20 @@ def wrapper(*args, **kwargs):
 
             # we notify the user that the interaction failed
             message = (
-                f"Sorry, we've run into an unexpected error. For help, please reach out to {conversation.incident.commander.individual.name}",
+                "Sorry, we've run into an unexpected error. For help please reach out to your Dispatch admins",
                 f" and provide them with the following token: {slack_interaction_guid}.",
             )
-            if conversation.incident.status != IncidentStatus.closed:
-                dispatch_slack_service.send_ephemeral_message(
-                    kwargs["slack_client"], channel_id, user_id, message
-                )
+
+            if conversation:
+                if conversation.incident.status != IncidentStatus.closed:
+                    dispatch_slack_service.send_ephemeral_message(
+                        kwargs["slack_client"], channel_id, user_id, message
+                    )
             else:
-                dispatch_slack_service.send_message(
+                dispatch_slack_service.send_ephemeral_message(
```

Comment:
Does this work? I haven't tried to DM an ephemeral before. Wouldn't it make more sense to make the message persist in a DM?

Reply:
What was the rationale behind sending a DM? I was issue command in a central channel e.g. incident notifications but the errors were sent to a DM? This change sends errors to whichever conversation the command originated and is ephemeral to let the user know something has gone wrong (similar to how we send "working on" messages).